dev/breeze/src/airflow_breeze/commands/ci_image_commands.py

```diff
@@ -296,6 +297,7 @@ def get_exitcode(status: int) -> int:
 @option_debug_resources
 @option_dev_apt_command
 @option_dev_apt_deps
+@option_disable_airflow_repo_cache
```

Comment:
Comment: CI image build command did not handle --disable-airflow-repo-cache flag. It was only handled in PROD build.